src/Services/Attendance/Attendance.SharedKernel/Interfaces/IQueryHandler.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+
+namespace Attendance.SharedKernel.Interfaces
+{
+    public interface IQueryHandler<TQuery, TQueryResponse>
+    {
+        TQueryResponse Handle(TQuery query);
```

Comment:
Remove this one. Also if you are using interface then you are forcing the user. Rather use abstract class if you wanna give the consumer options